docs/source/getting-started/app-architecture.rst

```diff
@@ -86,6 +86,13 @@ That is what the ``Follow`` choice does: after checking some preconditions, it a
 
 More detailed information on choices can be found in :doc:`our docs </daml/reference/choices>`.
 
+Finally, the ``User.daml`` file contains the Alias template that manages the link between user ids and aliases:
```

Comment:
I think we should tie this back to the public party and explain that we set it as observer.

Reply:
Something like this? "Here the public party created by the ``Setup.daml`` file is made an observer to the contract linking the user and alias. This shares the alias with everyone."